apps/console/src/extensions/components/authenticators/sms-otp/sms-otp-authenticator.tsx

```diff
@@ -93,27 +89,14 @@ export const SmsOTPAuthenticator: FunctionComponent<SmsOTPAuthenticatorInterface
         return !disabledFeatures?.includes("choreoAsSMSProvider");
     }, [ featureConfig ]);
 
-    const [ isReadOnly, setIsReadOnly ] = useState<boolean>(
-        isChoreoEnabledAsSMSProvider ||
-        !hasRequiredScopes(
-            featureConfig?.identityProviders, featureConfig?.identityProviders?.scopes?.update, allowedScopes) ||
-        isSubOrganization()
-    );
+    const isReadOnly: boolean = isChoreoEnabledAsSMSProvider
+        || !hasRequiredScopes(
+            featureConfig?.identityProviders, featureConfig?.identityProviders?.scopes?.update, allowedScopes
```

Comment:
if possible pass the orgnization type for `hasRequiredScopes` as well.

Reply:
Fixed with [98970b8](https://github.com/wso2/identity-apps/pull/5722/commits/98970b8eef5ba644d0f99e7b4fe91ac004efddc0)